cloudigrade/account/tests/test_util.py

```diff
@@ -0,0 +1,145 @@
+"""Collection of tests for utils in the account app."""
+import random
+import uuid
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from account import AWS_PROVIDER_STRING, util
+from account.models import AwsAccount, AwsMachineImage
+from util import aws
+from util.tests import helper as util_helper
+
+
+def mock_publish(body, exchange, routing_key, declare):
```

Comment:
Instead of creating a throwaway function, you could mock the `return_value` of a `Mock` object in your test.